core/src/agent/state.rs

```diff
@@ -284,7 +284,7 @@ pub mod tests {
         );
 
         assert_eq!(
-            "{\"header\":{\"entry_type\":\"testEntryType\",\"time\":\"\",\"next\":null,\"entry\":\"QmbXSE38SN3SuJDmHKSSw5qWWegvU7oTxrLDRavWjyxMrT\",\"type_next\":null,\"signature\":\"\"},\"entry\":{\"content\":\"test entry content\",\"entry_type\":\"testEntryType\"}}",
+            "{\"header\":{\"entry_type\":\"testEntryType\",\"timestamp\":\"\",\"prev\":null,\"entry_hash\":\"QmbXSE38SN3SuJDmHKSSw5qWWegvU7oTxrLDRavWjyxMrT\",\"entry_signature\":\"\",\"prev_same\":null},\"entry\":{\"content\":\"test entry content\",\"entry_type\":\"testEntryType\"}}",
```

Comment:
I don't think it is easier because its more of a regression test.
That string with the escape characters is a copy-past from the console's output.
So its actually easier keep it in this output format which has the escape characters, and copy-past again when we consciously change things.

Reply:
Why does the console output have escape characters? Also, what is the correlation to a regression test?

Reply:
Because this is what `serde_json::to_string()` outputs, which is called in the `assert_eq!`
When you copy-past the output into your test as the the thing to test against your are doing a regression test, i.e making sure the output stays the same.